lib/src/common/parameters/excluded_identifiers_list_parameter.dart

```diff
@@ -35,10 +68,11 @@ class ExcludedIdentifiersListParameter {
 
   /// Returns whether the target node should be ignored during analysis.
   bool shouldIgnore(Declaration node) {
-    final methodName = node.declaredElement?.name;
+    final declaredName = node.declaredElement?.name;
 
-    final excludedItem =
-        exclude.firstWhereOrNull((e) => e.methodName == methodName);
+    final excludedItem = exclude.firstWhereOrNull(
+      (e) => e.methodName == declaredName || e.className == declaredName,
+    );
```

Comment:
Check for declaration type before comparing. Right now we are checking classnames against method names and vice versa.